src/app/exams/exams-question.component.ts

```diff
@@ -10,7 +10,13 @@ import { uniqueId } from '../shared/utils';
   selector: 'planet-exam-question',
   templateUrl: 'exams-question.component.html',
   styles: [ `
-    .question-choices {
+    .survey-question-choices {
```

Comment:
you can leave the class as it is and just add another class which will just overwrite grid-template-columns